test/prototype_transforms_kernel_infos.py

```diff
@@ -126,6 +132,11 @@ def sample_inputs_horizontal_flip_mask():
         yield ArgsKwargs(image_loader)
 
 
+def sample_inputs_horizontal_flip_video():
```

Comment:
I intentionally only added a single `KernelInfo` for videos here, since we already have quite a few tests and videos will add even more. Thus, I'll go for 3. of #6622 first before adding more tests here. As explained above, the video kernels are a trivial wrapper around the image kernels for now, so unless I made a mistake while implementing the signature, there should be no new bugs.